source/contracts/reporting/InitialReporter.sol

```diff
@@ -86,6 +86,9 @@ contract InitialReporter is DelegationTarget, Ownable, BaseReportingParticipant,
     }
 
     function forkAndRedeem() public onlyInGoodTimes returns (bool) {
+        if (!isDisavowed()) {
```

Comment:
This only catches the case in which forkAndRedeem is called twice on the forking market's initial reporter?